src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig

```diff
@@ -1,5 +1,10 @@
 <div class="block-exception">
     <div class="block-exception-detected clear-fix">
+        <div class="support">
+            <span>
```

Comment:
That can also be done when using `<div class="support"><a ...></div>`, by changing `.sf-reset .support span` to `.sf-reset .support a`